topics/go/algorithms/strings/reverse/reverse_test.go

```diff
@@ -0,0 +1,29 @@
+package strings_test
+
+import (
+	strings "github.com/ardanlabs/gotraining/topics/go/algorithms/strings/reverse"
+	"testing"
+)
+
+const succeed = "\u2713"
+const failed = "\u2717"
+
+func TestReverseString(t *testing.T) {
+
+	revTests := []struct {
+		name   string
+		input  string
+		output string
+	}{
+		{"basic", "Hello World", "dlroW olleH"},
```

Comment:
Maybe add a string with 1, 2, odd and even number of characters. 

Create a string with Chinese characters.

Create a string with a character that uses two runes

Reply:
Completed